randovania/games/am2r/logic_database/The Tower.txt

```diff
@@ -929,20 +929,27 @@ Extra - minimap_data: [{'x': 33, 'y': 34}, {'x': 33, 'y': 35}, {'x': 33, 'y': 36
   > Event - Tester
       All of the following:
           Any of the following:
-              # Energy requirements
+              # Weapon Requirements
+              Missiles or Plasma Beam or Power Bombs or Super Missiles
```

Comment:
why plasma beam, does it circumvent its shell?

Reply:
Plasma allows the player to damage Testers life points directly. If you have plasma you can just aim up and spam beam shots.